src/structs.rs

```diff
@@ -0,0 +1,29 @@
+use macroquad::prelude::*;
```

Comment:
this is not needed

Reply:
Yeah I should've just used "macroquad::perlude::Rect"